ts/store/reducers/backendStatus.ts

```diff
@@ -481,11 +478,12 @@ export const isNewPaymentSectionEnabledSelector = createSelector(
 // on the icon in the headers of the top-level screens.
 // It will be possible to delete this control and all the code it carries
 // it carries when isNewPaymentSectionEnabledSelector and
-// showBarcodeScanSection will be deleted
+// isNewScanSectionLocallyEnabled will be deleted
 export const isSettingsVisibleAndHideProfileSelector = createSelector(
   isNewPaymentSectionEnabledSelector,
-  isNewPaymentSectionEnable =>
-    isNewPaymentSectionEnable && showBarcodeScanSection
+  isNewScanSectionLocallyEnabledSelector,
+  (isNewPaymentSectionEnabled, isNewScanSectionLocallyEnabled) =>
+    isNewPaymentSectionEnabled && isNewScanSectionLocallyEnabled
```

Comment:
Do you think it's already useful to use `isNewPaymentSectionEnabled` inside this selector? if I am not mistaken, this FF is always true, correct? So, I think we can delete the isSettingsVisibleAndHideProfileSelector selector and use only the `isNewScanSectionLocallyEnabled`, what do you think?

Reply:
From product & design the remote feature flag of the new payment section is still required until we reach 90% of users with the latest version of the app with the new endpoints (`2.65.0.0`), so by requirements currently it can't be removed. (It'll be removed as soon as we reach that milestone)

Reply:
okok perfect!